eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java

```diff
@@ -77,11 +77,31 @@ public final class BinarizeMojo extends SafeMojo {
     @Override
     public void exec() throws IOException {
         new Moja<>(BinarizeParseMojo.class).copy(this).execute();
-        for (final File project: targetDir.toPath().resolve("Lib").toFile().listFiles()) {
-            if (project.isDirectory() && project.toPath().resolve("Cargo.toml").toFile().exists()) {
-                this.build(project);
-            }
-        }
+        final int total = new SumOf(
+            new Threads<>(
+                Runtime.getRuntime().availableProcessors(),
+                new Mapped<>(
+                    project -> () -> {
```

Comment:
@levBagryansky What do you think if we move that lambda into a separate method? Maybe the code in this case will look simpler. In other words, I would separate the logic on two separate parts:
1. Thread management and mapping
2. Actual task (building a rust project)

Reply:
@volodya-lombrozo I think we can move `if-else` logic to `build` funtion then

Reply:
@volodya-lombrozo it pollutes the `build` method. Let's just move this `if-else` to separate method